TASK-4/UserFactory.cs

```diff
@@ -0,0 +1,19 @@
+namespace Persons;
+
+public class UserFactory
+{
+    public static User ReturnGeneratedTypeOfUser()
+    {
+        Console.WriteLine("Введите тип пользователя: Employee или Candidate");
+        var typeOfUser = Console.ReadLine();
+
+        if (typeOfUser == "Employee")
+        {
+            return (User)new Employee();
+        }
+        else
+        {
+            return (User)new Candidate();
+        }
+    }
```

Comment:
User Factory should generate and return a list of users depending on the user type. Create a method that accepts a user type and returns a list

Reply:
Done. In addition, I've created another method which shows a report on new users in case this feature may be useful in the future.